manifest.json

```diff
@@ -6,6 +6,7 @@
     "get-message-service-bus-topic-peek-lock-abandon",
     "get-message-service-bus-topic-peek-lock-renew",
     "get-sales-order-status-sap-teams-bapi",
+    "SAP-BAPI-GoodsMovementCreate",
```

Comment:
template name has to be lowercase only, please see other names as reference

Reply:
changed to lowercase, also title of the folder